warp/src/raygun/mod.rs

```diff
@@ -880,7 +880,20 @@ impl Conversation {
 #[derive(Debug, Clone, Copy, Deserialize, Serialize, PartialEq, Eq, Hash)]
 pub enum GroupPermission {
     AddParticipants,
-    SetGroupName,
+    RemoveParticipants,
+    EditGroupInfo,
+    EditGroupVisual,
```

Comment:
I assume this is for the conversation images? If so lets name it `EditGroupImages`.